engine/src/main/java/org/destinationsol/game/SolCam.java

```diff
@@ -47,6 +48,7 @@
     private final Vector3 myTmpVec;
 
     private float myPrevHeroLife;
+    private Hull myPrevHeroHull;
```

Comment:
```suggestion
    private Hull previousHeroHull;
```
We are trying to get rid of all these `myX` variable names as well as the abbreviations.

Reply:
Okay, I thought I'll stay consistent with the naming